system/templates/rolls/preroll-dialog.hbs

```diff
@@ -0,0 +1,71 @@
+{{#*inline "statDescription"}}
+{{#if (eq Using.length 1)}}
+roll +{{lookup Using 0}}
+{{else}}
+roll {{Method}} of {{join Using delimiter=', '}} {{!-- TODO: i18n --}}
+{{/if}}
+{{/inline}}
+
+<form class="preroll-dialog">
+  {{#if move}}
+  <div>{{{enrichMarkdown move.data.data.Trigger.Text}}}</div>
+
+  {{#if move.data.data.Trigger.Options}}
+  <ul>
+    {{#each move.data.data.Trigger.Options}}
+    {{#if Text}}
+    <li>
+      {{ Text }}: <strong>{{>statDescription}}</strong>
+    </li>
+    {{/if}}
+    {{/each}}
+  </ul>
+  {{/if}}
+  {{/if}}
+
+  <div class="dice-roll ironsworn-roll">
+    {{{graphic}}}
+  </div>
+
+  {{#if action}}
+  <div class="form-group">
+    <label>{{localize 'IRONSWORN.Adds'}}</label>
+    <input type="number" step="1" name="adds" value="0" />
+  </div>
+  {{/if}}
+
+  {{!-- TODO: simple options from the move + actor assets --}}
+
+  <details style="padding: 5px; margin: 5px 0; border: 1px solid">
+    <summary>{{localize 'IRONSWORN.RollDialog.AdvancedRollOptions'}}</summary>
+
+    <div class="form-group">
```

Comment:
consider using `<fieldset>` instead for better semantics: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

Reply:
In a clean room, I'd say yes. But Foundry uses and provides this `form-group` class, and it's definitely a norm there. See the example for [the `radioBoxes` Handlebars helper](https://foundryvtt.com/api/HandlebarsHelpers.html). Much less impedance to use Foundry's happy path than invent our own form styling.